lib/classes/checkingaccount_class.rb

```diff
@@ -0,0 +1,47 @@
+module BankAccount
+
+  class CheckingAccount < Account
+    attr_reader :checks
+
+    #inherits passed in parameters from Account class
+    def initialize(id, initial_balance)
+    super(id, initial_balance)
```

Comment:
You don't actually need to pass in arguments to `super` if they are the same as the current method